docs/cn/deploy.md

```diff
@@ -1,62 +1,71 @@
 # 部署OpenMLDB
 
-### 部署zookeeper
```

Comment:
Please also help change all incorrect forms of zookeeper/Zookeeper, to the official name "ZooKeeper" throughout this document. Thanks~